nemoguardrails/embeddings/providers/openai.py

```
@@ -46,11 +46,11 @@ def __init__(
         **kwargs,
     ):
         try:
-            import openai  # type: ignore
-            from openai import OpenAI  # type: ignore
+            import openai
+            from openai import OpenAI
         except ImportError:
             raise ImportError("Could not import openai, please install it with `pip install openai`.")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)